src/relay/transforms/simplify_expr.cc

```diff
@@ -941,6 +941,87 @@ class SimplifyDQArgSort : public SimplifyDQArgFunc {
   SimplifyDQArgSort() : SimplifyDQArgFunc("argsort") {}
 };
 
+/*! \brief Simplifying a * x * x + b * x * y + c * y * y to a * (x + p) * (y + q) */
```

Comment:
This line isn't correct---it should be `a(x+py)(x+qy)` (as per the PR description).